CONTRIBUTING.md

```diff
@@ -38,17 +54,85 @@ OS            | Supported tests
 OS X          | Android, iOS
 Windows       | Android
 
+### Test descriptions
+
+The tests first build the app.
+
+They then check if the required emulators are currently running.
+
+If they are not, then it attempts to start an Android emulator named ```emulator``` (if one exists) and an iOS simulator named ```iPhone 6s (9.3)```. If these emulators don't exist on your computer, the task will exit and the tests will fail.
```

Comment:
first available simulator would be really old since iOS installs a ton of simulators when u install xcode. i'll rewrite this code to grab the newest.